Src/AutoFixture/RandomNumericSequenceGenerator.cs

```diff
@@ -56,7 +56,13 @@ public RandomNumericSequenceGenerator(params long[] limits)
                 throw new ArgumentException("The limit must be a sequence of two or more integers.");
             }
 
-            this.limits = limits;
+            for (var i = 0; i < limits.Count() - 1; i++)
+            {
+                if (limits[i + 1] < limits[i])
+                    throw new ArgumentException("The limit must be an increasing sequence.");

```

Comment:
It looks like it is LOL!  I didn't notice because I forked the main branch and missed the pull request addressing it.  Apologies.